### ISSUE TITLE
Typo on /etc/msmtprc

### ISSUE THREAD

**Issue by rhatguy** [tiredofit/docker-db-backup#326]
### Summary

Typo in grep statement for /etc/msmptrc (pt and tp are reversed)

### Steps to reproduce
Configured db-backup to backup a mysql database using XZ compression to an S3 bucket and configured email notifications.

### What is the expected *correct* behavior?

Job should not error and backups should complete.

### Relevant logs and/or screenshots

```
2024-01-14.20:49:18 [ERROR] ** [01-unraid.travisandliz.com__ALL] Moving of backup 'mysql_guacamole_db_unraid_20240114-204916.sql.xz' reported errors
grep: /etc/msmptrc: No such file or directory

**Comment by rhatguy** [comment]
Issue seems to be at line 1391 of 
install/assets/functions/10-db-backup

**Comment by tiredofit** [comment]
Thanks! This is resolved in `4.0.35` 
https://github.com/tiredofit/docker-db-backup/commit/f7f72ba2c1c7b82aafaed1d8c97b37230fcd074f
